### DISCUSSION TITLE
Browsersync LIVE (alpha)

### DISCUSSION THREAD

**Discussion by shakyShane** [BrowserSync/browser-sync#2069] [Announcements]
re: Browsersync LIVE (alpha) https://browsersync.io/live

**Comment by freinbichler** [comment]
Sounds cool, can we have more details? The article does not make it clear if this is an independent version or a future update? When can we try it? When will it be released?

**Reply by shakyShane** [reply]
Coming up with examples/demos over this weekend - I posted this here to give me the motivation, thanks for commenting :)

**Reply by shakyShane** [reply]
> does not make it clear if this is an independent version or a future update

It's a separate tool - but with a lot of overlap/improvements. I care so deeply about the existing users of Browsersync that I don't want to disrupt them with any large breaking changes :)

**Comment by ajiho** [comment]
Do you have any new information to disclose? I like browser sync, but it often makes my CPU very high and sometimes crashes. If Browser sync LIVE could optimize in this area, I would be particularly happy

**Comment by trymeouteh** [comment]
When will this tool be in stable release? And will it replace the original browser sync tool?
